pkg/cmd/workspace/code.go

```diff
@@ -66,7 +66,12 @@ var CodeCmd = &cobra.Command{
 		} else {
 			workspace, err = apiclient_util.GetWorkspace(args[0])
 			if err != nil {
-				log.Fatal(err)
+				if err.Error() == "404 page not found" || err.Error() == "failed to get workspace: workspace not found"{
```

Comment:
As I said in the previous comment, you need to check if the error contains `workspace.ErrWorkspaceNotFound`. Hard matching this is fragile

Reply:
ok. now i got it

Reply:
see the changes.
![image](https://github.com/user-attachments/assets/1969fa1e-452d-4a61-a95c-12a6cf93a634)
I tried using errors.Is but didn't worked for some reasons

Reply:
By the way i used "github.com/daytonaio/daytona/pkg/server/workspaces"     workspaces it had many times of errors

Reply:
Found out that error wrapping was not done properly therefore I used %w instead of %s in order to unwrap it and use errors.Is

Reply:
@Divanshu-Grover thanks for that. Do you mind opening a new PR where you would change `%s` to `%w` everywhere in the project where errors are logged?

Reply:
ok. but do then i have to remove these changes from here

Reply:
No need, you can leave this here. We'll merge this today probably.